speed-dreams/source-2.2.3/src/modules/userinterface/textonly/textonly.cpp

```diff
@@ -137,7 +137,8 @@ bool LoadReplayConfiguration(GfRaceManager*& p_selRaceMan) {
     InterventionType interventionType = static_cast<InterventionType>(GfParmGetNum(replaySettingsHandle, PATH_INTERVENTION_TYPE, KEY_SELECTED, nullptr, INTERVENTION_TYPE_NO_SIGNALS));
     SMediator::GetInstance()->SetInterventionType(interventionType);
 
-    SMediator::GetInstance()->SetMaxTime(-1);
+    int maxTime = static_cast<int>(GfParmGetNum(replaySettingsHandle, PATH_MAX_TIME, KEY_MAX_TIME, nullptr, -1));
```

Comment:
LoadReplayConfiguration() is missing the SIMULATED DRIVING ASSISTANCE comment above the @brief